wagtail/admin/auth.py

```diff
@@ -13,9 +14,17 @@
 from wagtail.admin import messages
 from wagtail.log_actions import LogContext
 from wagtail.permission_policies.pages import PagePermissionPolicy
+from wagtail.utils.deprecation import RemovedInWagtail60Warning
 
 
 def users_with_page_permission(page, permission_type, include_superusers=True):
```

Comment:
Pretty sure this function doesn't really belong in here. This module is more concerned with view restrictions, e.g. view decorators or checks in `dispatch()`.

It's only ever used in `send_moderation_notification`, which in turn is only used by the legacy moderation system.

We could also remove this outright if desired.